deploy-data-tokeninzation-solution-v2.sh

```diff
@@ -57,6 +59,11 @@ gcloud projects add-iam-policy-binding ${PROJECT_ID} --member="serviceAccount:${
 
 # trigger the cloud build script to create DLP templates
 gcloud builds submit . --config dlp-demo-part-2-dlp-template.yaml --substitutions _KEK_CONFIG_FILE=gs://${DATA_STORAGE_BUCKET}/${KEK_FILE_NAME},_GCS_BUCKET_NAME=gs://${DATA_STORAGE_BUCKET},_API_KEY=$(gcloud auth print-access-token)
+
+# create Pub/Sub topic and enable GCS notification to the pub/sub topic
```

Comment:
Please add changes related to this in readme as well

Reply:
Will be taken after readme is revamped. Tracking via http://b/289161938